api/app/serializers/spree/v2/storefront/store_serializer.rb

```diff
@@ -10,6 +10,13 @@ class StoreSerializer < BaseSerializer
 
         has_many :menus
         has_one :default_country, serializer: :country, record_type: :country, id_method_name: :default_country_id
+
+        attribute :favicon_path do |store|
+          if store.favicon_image.attached?
+            url_helpers = Rails.application.routes.url_helpers
+            url_helpers.rails_representation_path(store.favicon, only_path: true)
```

Comment:
@damianlegawiec I'm not sure about use cases, but do you think that relative path will be enough?

Reply:
@mad-eel it will be fine! :)